### PR TITLE
Unify sequence table, show visits, virtualize

### PR DESCRIPTION
**Pull request by rpiaggio**
The sequence table now:
- Is a single table (it was 2 tables before, for acquisition and science).
- Virtualized.
- Shows visits.

### CODE REVIEW

**Comment by rpiaggio** [comment]
There are some changes related to the core upgrade.

File: modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTableDefs.scala [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    List(
+      SequenceColumns.headerCell(HeaderColumnId, ColDef).setColumnSize(ColumnSizes(HeaderColumnId)),
+      column(
+        BreakpointColumnId,
+        "",
+        cell =>
+          cell.row.original.value.toOption.map: stepRow =>
+            val step: SequenceRow[DynamicConfig] = stepRow.step
+            val stepId: Option[Step.Id]          = step.id.toOption
+            // val canSetBreakpoint =
```
Comment (cquiroz) [inline modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTableDefs.scala:152]: Commented code

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTableDefs.scala (hunk 1 above)
Comment (rpiaggio) [reply modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTableDefs.scala:152]: This code was already commented, it was just moved to another file. It's code from seqexec for which we still have to migrate the logic.

**Pull request merged**